src/lib/accessibility/Behaviors/Popup/PopupBehavior.ts

```diff
@@ -0,0 +1,34 @@
+import { Accessibility } from '../../interfaces'
+import * as keyboardKey from 'keyboard-key'
+
+const isButton = props => {
```

Comment:
I just noticed that you use this to add some attributes to non-buttons; is that really necessary? I don't see any problem in overriding attributes here

Reply:
you are right that this check looks not nice, haven't yet come up with a better idea.
And yes, it's important to set these attributes ```role=button``` and ```tabindex=0```, in case it's not a focusable element, for e.g. when user will use just ```div``` as a trigger.

Reply:
We can set it, but then we may break user's implementation if they need anything else.
To be safer, we need to put ```role=button```  and ```tabIndex=0```, if these attributes weren't set already by a user or the element is not focusable (which are in fact 'input', 'button', 'a').
I've made changes to that part, let me know what do you think.